addons/account/models/account_payment.py

```diff
@@ -483,21 +483,28 @@ def _compute_is_internal_transfer(self):
                                            and payment.partner_id == payment.journal_id.company_id.partner_id \
                                            and payment.destination_journal_id
 
-    @api.depends('available_payment_method_line_ids')
+    @api.depends('available_payment_method_line_ids', 'partner_id')
     def _compute_payment_method_line_id(self):
         ''' Compute the 'payment_method_line_id' field.
         This field is not computed in '_compute_payment_method_line_fields' because it's a stored editable one.
         '''
+        inbound_payment_method_id = self.partner_id.property_inbound_payment_method_line_id.id
```

Comment:
In the context of this compute i don't think we can have multiple partner but since it's just an assignation of value i think we can move it in the loop :smile: